TritonInferenceServer/utilities/stats.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020-2021 MobiledgeX, Inc. All rights and licenses reserved.
+# MobiledgeX, Inc. 156 2nd Street #408, San Francisco, CA 94105
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+
+class RunningStats:
+
+    def __init__(self):
+        self.n = 0
+        self.current = 0
+        self.old_m = 0
+        self.new_m = 0
+        self.old_s = 0
+        self.new_s = 0
```

Comment:
Can you use clearer names here? It's a little confusing what these variables mean. eg. self.old_m -> self.old_mean and self.old_s -> self.old_sum_squares

Reply:
Coming from a math guy, this comment carries more weight than usual.